### PR TITLE
Several pecan fixes

### PR DESCRIPTION
**Pull request by javierarce**
This PR include several fixes:

* Enables Torque analysis.
* Returns the `null_count`.
* Updates the static Image library to allow using an `api_key`.
* Adds a missing color scale.
* Enables the use of the `CDB_DistType` method.

### CODE REVIEW

**Comment by javierarce** [comment]
@javisantana: can you review this, please?

File: src/api/cartocss.js [hunk 1]
```diff
@@ -140,22 +170,30 @@ function guessMap(sql, tableName, column, stats) {
...
-    }
+  } else if (stats.type === 'date') {
+    wizard = "torque";
+    css = CSS.torque(stats, tableName);
+  }
 
   if (css) {
-    return { sql: sql, css: css, geometryType: geometryType, column: columnName, bbox: bbox, stats: stats, type: type, wizard: wizard  };
+    if (metadata) {
+      return { sql: sql, css: css, metadata: metadata, geometryType: geometryType, column: columnName, bbox: bbox, stats: stats, type: type, wizard: wizard  };
```
Comment (javisantana) [inline src/api/cartocss.js]: I'd not call it wizard, in this context is more like "visualizationType" than wizard

**Comment by javisantana** [comment]
apart of my comment and missing tests i don't see any problem

File: src/api/cartocss.js (hunk 1 above)
Comment (javierarce) [inline src/api/cartocss.js]: :+1: makes sense

**Comment by javierarce** [comment]
retest this please

**Pull request merged**